lib/gui/modules/analytics.js

```diff
@@ -39,6 +41,42 @@ const analytics = angular.module(MODULE_NAME, [
   require('../models/settings')
 ]);
 
+/**
+ * @summary Create an object copy with all absolute paths replaced with the path basename
+ * @function
+ * @private
+ *
+ * @param {Object} object - original object
+ * @returns {Object} transformed object
+ *
+ * @example
+ * const anonymized = hidePaths({
+ *   path1: '/home/john/rpi.img',
+ *   simpleProperty: null,
+ *   nested: {
+ *     path2: '/home/john/another-image.img',
+ *     path3: 'yet-another-image.img',
+ *     otherProperty: false
+ *   }
+ * });
```

Comment:
With the code as it is currently, I guess it'd also 'anonymize' `/dev/sdb` -> `sdb` and `\\\\.\\PHYSICALDRIVE1` -> `PHYSICALDRIVE1` ?
(which of course isn't a problem, if we know to expect that when looking at the logs)